packages/eventindexer/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
+services:
+  db:
+    image: mysql:8.0
+    cap_add:
+      - SYS_NICE
+    restart: always
+    environment:
+      - MYSQL_DATABASE=eventindexer
+      - MYSQL_ROOT_PASSWORD=passw00d
```

Comment:
password in this file?

Reply:
@SamiAlHassan Please update the PR since the code in the main branch has been changed. Thanks.